sdk/communication/azure-communication-phonenumbers/swagger/README.md

```diff
@@ -190,4 +188,31 @@ directive:
     transform: >
       $["properties"]["localizedName"].readOnly = true;
       $["properties"]["countryCode"].readOnly = true;
+```
+
+### Removed Property error from PhoneNumberSearchResult
+``` yaml
+directive:
+  where-model: PhoneNumberSearchResult
+  remove-property: error
+```
+
+### Removed Property errorCode from PhoneNumberSearchResult
+``` yaml
+directive:
+  where-model: PhoneNumberSearchResult
+  remove-property: errorCode
+```
+
+### Removed Property phoneNumberSource from PurchasedPhoneNumber
+``` yaml
+directive:
+  where-model: PurchasedPhoneNumber
+  remove-property: phoneNumberSource
+```
+
+# Removed Model Error
+``` yaml
+directive:
+  - remove-model: Error
 ```
```

Comment:
Could we add the same suppressions from the .NET SDK to prevent cherry picker generation? This is important because the generated client counts towards our coverage, but it is not used at all.

https://github.com/Azure/azure-sdk-for-net/blob/03d46a5e0a868fa42d4e25c2bd8065932a209e0e/sdk/communication/Azure.Communication.PhoneNumbers/src/autorest.md?plain=1#L44-L54